src/types/CodepointWidthDetector.cpp

```diff
@@ -310,13 +310,21 @@ namespace
     };
 }
 
+// Routine Description:
+// - Constructs an instance of the CodepointWidthDetector class
+CodepointWidthDetector::CodepointWidthDetector() noexcept :
+    _fallbackCache{},

```

Comment:
both of these should automatically get default-constructed for us, and we should be able to default our constructor.

Reply:
I think I did that and it carped so I did this.